.pipelines/OneBranch.Official.yml

```diff
@@ -37,8 +37,16 @@ extends:
       enabled: false
     
     globalSdl:
+      isNativeCode: true
```

Comment:
What does isNativeCode do? I'm having a hard time finding this in the docs

Reply:
Replied offline with a docs link.